src/extension.ts

```diff
@@ -143,6 +143,12 @@ export async function activate(context): Promise<extensionapi.ExtensionAPI> {
         registerCommand('extension.vsKubernetesCreate',
             maybeRunKubernetesCommandForActiveWindow.bind(this, 'create', "Kubernetes Creating...")
         ),
+        registerCommand('extension.vsKubernetesCreateFile',
+            async (uri: vscode.Uri) => kubectlUtils.createResourceFromUri(uri, kubectl)),
```

Comment:
You're not using `await` in the lambda body so you don't need the lambda to be `async`.

Reply:
done.